src/server/api/routers/hackers.ts

```diff
@@ -16,7 +16,13 @@ export const hackerRouter = createTRPCRouter({
 				),
 		)
 		.query(async ({ ctx, input }) => {
-			if (ctx.session.user.role === "HACKER"){
+			const userId = ctx.session.user.id;
+			const user = await ctx.prisma.user.findUnique({
+				where: {
+					id: userId,
+				},
+			});
+			if (user.role !== Role.HACKER){
```

Comment:
The build is failing because you need to check that a user was returned before you access the `role` property.
```js
if (!user) {
    throw new Error("User not found");
}
		```